src/xsdba/testing.py

```diff
@@ -0,0 +1,130 @@
+"""Testing utilities for xsdba."""
+
+from __future__ import annotations
+
+import collections
+import hashlib
+import logging
+import os
+import warnings
+from pathlib import Path
+from urllib.error import HTTPError, URLError
+from urllib.parse import urljoin, urlparse
+from urllib.request import urlopen, urlretrieve
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from platformdirs import user_cache_dir
+from scipy.stats import gamma
+from xarray import open_dataset as _open_dataset
+
+from xsdba.utils import equally_spaced_nodes
+
+__all__ = ["nancov", "test_timelonlatseries", "test_timeseries"]
+
+# keeping xclim-testdata for now, since it's still this on gitHub
+_default_cache_dir = Path(user_cache_dir("xclim-testdata"))
+
+# XC
+TESTDATA_BRANCH = os.getenv("XCLIM_TESTDATA_BRANCH", "main")
+"""Sets the branch of Ouranosinc/xclim-testdata to use when fetching testing datasets.
+
+Notes
+-----
+When running tests locally, this can be set for both `pytest` and `tox` by exporting the variable:
+
+.. code-block:: console
+
+    $ export XCLIM_TESTDATA_BRANCH="my_testing_branch"
+
+or setting the variable at runtime:
+
+.. code-block:: console
+
+    $ env XCLIM_TESTDATA_BRANCH="my_testing_branch" pytest
+
+"""
```

Comment:
The new system uses pooch. Will need to update this here.

Reply:
since the testing simply relies on xclim right now, I think we can simply remove everything related to this

Reply:
This has been updated. Good enough for now.